### PR TITLE
Aggregation should be a list

### PR DESCRIPTION
**Pull request by rakshith91**
Fixes https://github.com/Azure/azure-sdk-for-python/issues/19251

### CODE REVIEW

File: sdk/monitor/azure-monitor-query/tests/test_metrics_client.py [hunk 1]
```diff
@@ -15,9 +16,13 @@ def _credential():
 def test_metrics_auth():
     credential = _credential()
     client = MetricsQueryClient(credential)
-    # returns LogsQueryResults 
-    response = client.query(os.environ['METRICS_RESOURCE_URI'], metric_names=["PublishSuccessCount"], timespan='P2D')
-
+    response = client.query(
+        os.environ['METRICS_RESOURCE_URI'],
+        metric_names=["MatchedEventCount"],
```
Comment (xiangyan99) [inline sdk/monitor/azure-monitor-query/tests/test_metrics_client.py:21]: Is it by design to change it from "PublishSuccessCount" to "MatchedEventCount"?

**Review by xiangyan99 — APPROVED** [review]
Nothing insane.

File: sdk/monitor/azure-monitor-query/tests/async/test_metrics_client_async.py [hunk 2]
```diff
@@ -16,9 +16,13 @@ def _credential():
...
-
+    response = await client.query(
+        os.environ['METRICS_RESOURCE_URI'],
+        metric_names=["MatchedEventCount"],
+        start_time=datetime(2021, 6, 21),
+        duration='P1D',
+        aggregation=['Count']
+        )
     assert response is not None
     assert response.metrics is not None
```
Comment (rakshith91) [inline sdk/monitor/azure-monitor-query/tests/async/test_metrics_client_async.py:27]:
```suggestion
    assert response
    assert response.metrics
```
```suggestion
    assert response is not None
    assert response.metrics is not None
```

File: sdk/monitor/azure-monitor-query/tests/test_metrics_client.py (hunk 1 above)
Comment (rakshith91) [reply sdk/monitor/azure-monitor-query/tests/test_metrics_client.py:21]: the metric name publishe success count does not have any aggregations supported - these metric names are defined by each service

File: sdk/monitor/azure-monitor-query/tests/test_metrics_client.py [hunk 3]
```diff
@@ -15,9 +16,13 @@ def _credential():
...
+        start_time=datetime(2021, 6, 21),
+        duration='P1D',
+        aggregation=['Count']
+        )
     assert response is not None
     assert response.metrics is not None
```
Comment (rakshith91) [inline sdk/monitor/azure-monitor-query/tests/test_metrics_client.py:28]:
```suggestion
    assert response
    assert response.metrics

```

**Pull request merged**